checker/src/main/java/org/checkerframework/checker/mustcall/CreatesObligationElementSupplier.java

```diff
@@ -0,0 +1,29 @@
+package org.checkerframework.checker.mustcall;
+
+import javax.lang.model.element.ExecutableElement;
+import org.checkerframework.dataflow.cfg.node.MethodInvocationNode;
+import org.checkerframework.framework.type.GenericAnnotatedTypeFactory;
+import org.checkerframework.javacutil.AnnotationUtils;
+
+/**
```

Comment:
It would be helpful to state the reason this interface exists.  In this pull request, only one class implements it.